variables.tf

```diff
@@ -102,6 +102,18 @@ variable "worker_pool_id" {
   description = "ID of the the worker pool"
 }
 
+variable "cloudwatch_agent_config_version" {
+  type        = string
+  default     = "2"
```

Comment:
Maybe it would be safer to leave the default null/0 for this, and only run the logic if someone explicitly sets this variable? It means that if someone is using an old AMI that doesn't have the new files bundled, it won't break if they use the latest version of this module.